backend/de.metas.acct.base/src/main/java-legacy/org/compiere/acct/FactLineBuilder.java

```diff
@@ -568,12 +575,26 @@ public FactLineBuilder bpartnerIdIfNotNull(final BPartnerId bpartnerId)
 		}
 	}
 
+	@NonNull
+	public FactLineBuilder bpartnerLocationId(@Nullable final BPartnerLocationId bPartnerLocationId)
+	{
+		assertNotBuild();
+		this.bPartnerLocationId = bPartnerLocationId;
```

Comment:
suggestion: in case bPartnerLocationId != null wdyt about setting this.bpartnerId too?
anyways the bpartnerId and bpartnerLocationId shall match...